packages/cli/src/ui/components/messages/ToolConfirmationMessage.tsx

```diff
@@ -250,7 +250,10 @@ export const ToolConfirmationMessage: React.FC<
     bodyContent = (
       <Box flexDirection="column" paddingX={1} marginLeft={1}>
         <Text color={theme.text.link}>
```

Comment:
remove the outer text object as it isn't needed.

Reply:
done